.github/workflows/ci-cd.yml

```diff
@@ -98,4 +98,4 @@ jobs:
         poetry-version: 1.1.13
 
     - name: Build and push package to PYPI
-      run: poetry run nox -s release -- --username=__token__ --password=${{ secrets.pypi_token }}
+      run: POETRY_PYPI_TOKEN_PYPI="${{ secrets.pypi_token }}" poetry run nox -s release
```

Comment:
Use env property of GitHub action

Reply:
Can we try this, I saw some issues in the poetry repos for the env variable

Reply:
Can you elaborate what you mean exactly?